rbc/omniscidb.py

```diff
@@ -770,6 +770,8 @@ def retrieve_targets(self):
                 tname, value = tname_value.split(':')
                 null_values_asint[tname] = int(value)
                 dtype = tname
+                if dtype == 'text_encoding_dict':
+                    continue
```

Comment:
This is unnecessary as omniscidb server does not send `text_encoding_dict` null value anymore.

Btw, if omniscidb would need to send `text_encoding_dict` null value, it should use the format `<typename><bitwidth>`, e.g. `test_encoding_dict32`.